docs/en/glossary/glossary.asciidoc

```diff
@@ -454,6 +477,16 @@ depending on the characteristics of the event. Popular filter plugins include
 grok, mutate, drop, clone, and geoip. Filter stages are optional.
 //Source: Logstash
 
+[[glossary-fleet]] Fleet::
+A web-based UI in {kib} used to add and manage {agent}s.
```

Comment:
Fleet provides a way to centrally manage {agent}s at scale. There are two parts. The Fleet app in {kib} provides a web-based UI to add and remotely manage agents. The {fleet-server} provides the backend service to manage agents.

Reply:
```suggestion
Fleet provides a way to centrally manage {agent}s at scale. There are two parts: The Fleet app in {kib} provides a web-based UI to add and remotely manage agents, while the {fleet-server} provides the backend service that manages agents.
```